coinpaprika/adapter.js

```diff
@@ -87,7 +88,9 @@ const execute = (input, callback) => {
     case ENDPOINT_PRICE:
       return price(jobRunID, input, callback)
     case ENDPOINT_MKTCAP:
-      return globalMarketCap(jobRunID, input, callback)
+      return global(jobRunID, input, 'market_cap_usd', callback)
+    case ENDPOINT_DOMINANCE:
+      return global(jobRunID, input, 'bitcoin_dominance_percentage', callback)
```

Comment:
We should probably take the coin as a a parameter here as well.

I know they don't track any other coin's dominance here, but it will save us work in the future if they change that and we need something other than BTC's dominance.

Since all other adapters require the coin's symbol (e.g. "BTC"), you can convert this quickly like this:

```javascript
const convert = {
  BTC: 'bitcoin'
}
```

Reply:
I made these more dynamic by adding a parameter on the `global` helper for if the coin should be utilized to key into the payload. It does add the caveat of having the coin required for getting market capitalization. As seen in the test I just pass in 'USD' for that example.

Let me know if this is in line with what you were thinking. I could also make `coin/market/ect.` an optimal param there.